jenkins-config/managed-config-files/globalMavenSettings/global-settings/content

```diff
@@ -0,0 +1,257 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
You can rip out all of the boiler plate comments. They aren't needed

Reply:
Thank you .. done..